### PR TITLE
Fixed trailing white space stripping function

### PR DESCRIPTION
**Pull request by ngiangre**
# ⚠️ __IMPORTANT: Please do not create a Pull Request without creating an Issue first.__

All changes need to be discussed before proceeding. Failure to do so may result in the pull request being rejected.

Before submitting a pull request, please be sure to review:
- Our Code of Conduct: https://github.com/COVID-19-electronic-health-system/.github/blob/master/CODE_OF_CONDUCT.md
- Our Contributing Guidelines: https://github.com/COVID-19-electronic-health-system/.github/blob/master/CONTRIBUTING.md
- Our Support documentation: https://github.com/COVID-19-electronic-health-system/.github/blob/master/SUPPORT.md

-----

## Please include the issue number the pull request fixes by replacing YOUR-ISSUE-HERE in the text below.

Fixes #YOUR-ISSUE-HERE

## Summary

I had the wrong function to replace trailing white space. It is fixed now. And that solves the concatenation problem with the first and second word in a sentence:

problem

Screen Shot 2020-05-13 at 12 13 29 PM

## Final Checklist

- [ ] For CoronaTracker, did you bump the version in `package.json`?
    - Update the second decimal for a major change
    - Update the third decimal for a minor change
    - Numbers can go past 9, e.g. `1.0.9` => `1.0.10`
    - For more info, read about [Semantic Versioning](https://semver.org/)
- [ ] Did you add any new tests as necessary?
- [ ] Is your PR rebased off the most current master?
- [ ] Have you squashed all commits? _(can be done at merge)_
- [ ] Did you use `yarn` not `npm`? _(important!)_
- [ ] Did you use Material-UI wherever possible?
- [ ] Did you run `yarn lint` on the code?
- [ ] Did you run all of your most recent changes locally to make sure everything is working?

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+numpy==1.16.2
+pandas==0.24.2
+nltk==3.4
```
Comment (AdhamAH) [inline requirements.txt:3]: We had a problem with this, it needs to be`nltk==3.4.5`, did you rebase before pushing?

Comment (ngiangre) [reply requirements.txt:3]: no - my mistake. Should I retract the PR, rebase, then create again?

Comment (AdhamAH) [reply requirements.txt:3]: you can rebase then push again same PR

Comment (ngiangre) [reply requirements.txt:3]: ok fixing conflicts now

**Comment by ngiangre** [comment]
I think I fixed it!

**Pull request merged**